python/looker_sdk/sdk/api31/methods.py

```diff
@@ -6057,7 +6080,7 @@ def search_folders(
     #
     # Boolean search params accept only "true" and "false" as values.
     #
-    # GET /groups/search -> Sequence[models.Group]
+    # GET /groups/search -> Sequence[models.GroupSearch]
```

Comment:
this feels like not a great change for a "stable" API :-(

Reply:
Agreed, but codegen is just a consumer. Should take up the issue with the endpoint owners.